# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/serviceworkers/ServiceWorkerPrivate.cpp
# Commit: 2e24d997d5de
# Full Hash: 2e24d997d5de727be8e44b522e57d7ff7db52486
# Author: Sandor Molnar <smolnar@mozilla.com>
# Date: 2025-07-16 21:45:35
# Regressor Bug: 1972117
# File Overlap Count: 1
# Description:
#   Revert "Bug 1972117: apply code formatting via Lando" for causing assertion failures @ MozPromise.h
#   
#   This reverts commit e76333b4d7fb2d323e797f12bde50ccc99a51a85.
#   
#   Revert "Bug 1972117 - Part 4: Clear notifications on service worker unregistration r=asuth"
# ==============================================================================

diff -r c257659aa2b4 -r 2e24d997d5de dom/serviceworkers/ServiceWorkerPrivate.cpp
--- a/dom/serviceworkers/ServiceWorkerPrivate.cpp	Wed Jul 16 19:15:43 2025 +0300
+++ b/dom/serviceworkers/ServiceWorkerPrivate.cpp	Wed Jul 16 19:18:03 2025 +0300
@@ -1050,7 +1050,8 @@
 }
 
 nsresult ServiceWorkerPrivate::SendNotificationClickEvent(
-    const IPCNotification& aNotification, const nsAString& aAction) {
+    const nsAString& aScope, const IPCNotification& aNotification,
+    const nsAString& aAction) {
   MOZ_ASSERT(NS_IsMainThread());
 
   ServiceWorkerNotificationClickEventOpArgs clickArgs;
@@ -1067,7 +1068,7 @@
 }
 
 nsresult ServiceWorkerPrivate::SendNotificationCloseEvent(
-    const IPCNotification& aNotification) {
+    const nsAString& aScope, const IPCNotification& aNotification) {
   MOZ_ASSERT(NS_IsMainThread());
 
   ServiceWorkerNotificationCloseEventOpArgs closeArgs;